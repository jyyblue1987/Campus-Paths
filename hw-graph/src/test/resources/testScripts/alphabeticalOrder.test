# To test if the graph will output the two nodes we add in alphabetical order

# Create a graph
CreateGraph A

# Add two nodes
AddNode A z
AddNode A x

# Print out the nodes in the graph along with their outgoing edges
ListNodes A
ListChildren A x
ListChildren A z