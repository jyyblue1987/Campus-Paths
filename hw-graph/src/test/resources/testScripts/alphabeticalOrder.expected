# To test if the graph will output the two nodes we add in alphabetical order

# Create a graph
created graph A

# Add two nodes
added node z to A
added node x to A

# Print out the nodes in the graph along with their outgoing edges
A contains: x z
the children of x in A are:
the children of z in A are: